[2023-05-13T21:44:31.726+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hypothesis2.select_data manual__2023-05-13T21:43:25.375728+00:00 [queued]>
[2023-05-13T21:44:31.744+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hypothesis2.select_data manual__2023-05-13T21:43:25.375728+00:00 [queued]>
[2023-05-13T21:44:31.746+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-05-13T21:44:31.775+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): select_data> on 2023-05-13 21:43:25.375728+00:00
[2023-05-13T21:44:31.785+0000] {standard_task_runner.py:57} INFO - Started process 2021 to run task
[2023-05-13T21:44:31.791+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'hypothesis2', 'select_data', 'manual__2023-05-13T21:43:25.375728+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/dag_hypothesis2.py', '--cfg-path', '/tmp/tmp_tunyzmv']
[2023-05-13T21:44:31.796+0000] {standard_task_runner.py:85} INFO - Job 75: Subtask select_data
[2023-05-13T21:44:31.890+0000] {task_command.py:410} INFO - Running <TaskInstance: hypothesis2.select_data manual__2023-05-13T21:43:25.375728+00:00 [running]> on host 9a745d3878c0
[2023-05-13T21:44:32.007+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hypothesis2' AIRFLOW_CTX_TASK_ID='select_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-13T21:43:25.375728+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-13T21:43:25.375728+00:00'
[2023-05-13T21:44:32.022+0000] {base.py:73} INFO - Using connection ID 'qiberry' for task execution.
[2023-05-13T21:44:32.028+0000] {logging_mixin.py:149} INFO - Postgres connect success
[2023-05-13T21:44:32.032+0000] {logging_mixin.py:149} INFO - ok data
[2023-05-13T21:44:32.033+0000] {logging_mixin.py:149} INFO - Jeep
[2023-05-13T21:44:32.035+0000] {logging_mixin.py:149} INFO - ok data
[2023-05-13T21:44:32.036+0000] {logging_mixin.py:149} INFO - Ford
[2023-05-13T21:44:32.038+0000] {logging_mixin.py:149} INFO - brand , middle_price , middle_days_on_market , has_accidents_percent
[2023-05-13T21:44:32.042+0000] {warnings.py:110} WARNING - /opt/***/dags/dag_hypothesis2.py:48: RuntimeWarning: invalid value encountered in double_scalars
  self.middle_days_on_market = np.sum(self.days_on_market) / self.days_on_market.size

[2023-05-13T21:44:32.044+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_hypothesis2.py", line 90, in select_data_py
    x.calculate_params()
  File "/opt/airflow/dags/dag_hypothesis2.py", line 49, in calculate_params
    self.has_accidents_percent = np.count_nonzero(self.has_accidents == True) / self.has_accidents.size
ZeroDivisionError: division by zero
[2023-05-13T21:44:32.058+0000] {taskinstance.py:1373} INFO - Marking task as FAILED. dag_id=hypothesis2, task_id=select_data, execution_date=20230513T214325, start_date=20230513T214431, end_date=20230513T214432
[2023-05-13T21:44:32.072+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 75 for task select_data (division by zero; 2021)
[2023-05-13T21:44:32.125+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-05-13T21:44:32.161+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
